packages/wrangler/src/inspect.ts

```diff
@@ -217,8 +217,20 @@ export default function useInspector(props: InspectorProps) {
 		setRetryRemoteWebSocketConnectionSigil((x) => x + 1);
 	}
 
-	/** A simple incrementing id to attach to messages we send to devtools */
-	const messageCounterRef = useRef(1);
+	/** A simple decrementing id to attach to messages we send to devtools.
+	 * We use negative numbers to avoid ID collisions with the DevTools frontend:
+	 * https://github.com/ChromeDevTools/devtools-frontend/blob/88d8530ef2aec2c629354842dd7e9e212a025bf2/front_end/core/protocol_client/InspectorBackend.ts#L311
+	 * `workerd` accepts singed `Int32` values for `id`, so this is fine:
+	 * https://github.com/cloudflare/workerd/blob/5d644d0189246a22eec31bfd7b7c58a4afe40e57/src/workerd/io/cdp.capnp#L376
+	 */
+	const _messageCounterRef = useRef(-1);
+	const _messageResolves = useRef<Map<
+		number,
+		(result: unknown) => void
+	> | null>(null);
```

Comment:
I saw the useEffect used later, is there a way we can do this without involving Ink or are we currently to bound to React still in this part of the codebase?